### DISCUSSION TITLE
Use `table.header()` in table with only one column

### DISCUSSION THREAD

**Discussion by lettuceDestroyer** [typst/typst#4017] [Q&A] [answered]
I'm trying to create a table with repeating headers for a table that only has one column.

What I tried is this:
```typst
#let date = datetime.today().display()

#show table.header : it => strong(it)

#figure(
  align(center,
)[#table(
      columns: 1,
      align: left,
      inset: 6pt,
      table.header([Date],[Text 1], [Text 2]),
      [#date],
      [#lorem(15)],
      [#lorem(30)],
    )],
    caption: [Journal from #date]
)
```

Which resulted in this:
![image](https://github.com/typst/typst/assets/110594766/8a433c42-01a1-418f-bd37-b7eb49b830a8)

What I excepted was this:
![image](https://github.com/typst/typst/assets/110594766/5bf0cbe8-025d-4575-a5df-9e6a9d4620a9)

On a side note: Would it be alright to open a feature request for this?

**Comment by PgBiel** [comment, marked as answer]
The table header cannot be split apart, otherwise we don't have a way to correctly repeat it.

The behavior you see is intended - specifying cells in a header is similar to not having the header there at all. Since there is just one column, each cell triggers a row break.

You could open a feature request, but I'd suggest just typesetting your table manually for the time being, as the feature you seek seems complex to design, let alone implement.

**Reply by lettuceDestroyer** [reply]
Funnily enough my problem is that I do not know how to repeat a specific cell when the table spans over multiple pages. Do you think maybe you'd be able to help me with that?

**Reply by PgBiel** [reply]
There is no way to do that other than through a header or footer. You will have to manually typeset your table (place and repeat cells in such a way that the same cells appear in each page).
